### ISSUE TITLE
Products not syncing

### ISSUE THREAD

**Issue by hakeem1223** [facebookincubator/facebook-for-woocommerce#326]
My products aren't syncing correctly. The first time I tried only 10 product synced correctly and the rest didn't show the pics. I then disconnected the facebook for Woo-commerce plugin and re-synced my products. Now there is only one product in my store and the number count somehow is a negative number for remaining products. I have a little over 700 products on my site. All required plugins are up to date and using the latest version of Wordpress.

Also my products are from amazon.

<img width="1440" alt="screen shot 2018-03-14 at 12 13 11 pm" src="https://user-images.githubusercontent.com/37356040/37381678-cb09ced8-2781-11e8-9e9b-a29d139829d7.png">

**Comment by dmitridr** [comment]
Hello,

Thanks for writing in.
I haven't seen this bug for a while, but I think we had a negative items bug way back in 1.5.x, could you confirm that you are using the latest version v1.8.1 and if not try upgrading using our [releases page](https://github.com/facebookincubator/facebook-for-woocommerce/releases)?

**Comment by hakeem1223** [comment]
Yes I'm using the latest version v1.8.1. Now the problem is I keep getting the gateway time out even after I delete the cookies and browsing history. The pixel is correct on the Facebook side and I even tried to create a new Facebook Page store and it does the same thing. I had no problem uploading product a few months back when I was using the DIVI for my Alidropship. Now I'm using woozone amazon plugin with a Kingdom theme. I'm send along the screen shot of my current installed plugins.



<img width="1440" alt="screen shot 2018-03-16 at 5 57 36 am" src="https://user-images.githubusercontent.com/37356040/37490874-538a0d32-28df-11e8-8ac5-7abc65fa6053.png">
<img width="1440" alt="screen shot 2018-03-16 at 5 57 29 am" src="https://user-images.githubusercontent.com/37356040/37490875-53af961a-28df-11e8-857c-2c03e7e5981e.png">
<img width="1440" alt="screen shot 2018-03-16 at 5 57 24 am" src="https://user-images.githubusercontent.com/37356040/37490876-53d917ba-28df-11e8-9e09-27a1801f465b.png">
<img width="1440" alt="screen shot 2018-03-16 at 5 57 17 am" src="https://user-images.githubusercontent.com/37356040/37490877-5402cb14-28df-11e8-950a-6d315f429693.png">
<img width="1440" alt="screen shot 2018-03-16 at 5 57 10 am" src="https://user-images.githubusercontent.com/37356040/37490878-542d742c-28df-11e8-82d0-d20a3b5636b0.png">

**Comment by hakeem1223** [comment]
I tried to connect to intergration through woocommerce settingd and this keeps happening.
<img width="1440" alt="screen shot 2018-03-16 at 7 46 35 am" src="https://user-images.githubusercontent.com/37356040/37494929-47a08172-28ee-11e8-8290-4d408fd6ed6c.png">

**Comment by dmitridr** [comment]
Hello, 

Thanks for confirming. Yeah it seems likely to me that, especially if you could use the plugin just fine before, the problem is being caused by some new plugin that you may have, perhaps the one you mentioned, the woozone amazon plugin.

Could you try disabling the plugin(s) you think are related to the cause temporarily, then doing the sync? If that plugin you mention is the cause, the sync should work. Thanks for the screenshots of your plugin list, but I don't recognize any of them as ones that we know conflict with ours, this may be one we don't know about.

**Comment by hakeem1223** [comment]
Ok. I disabled woozone plugin and switched my theme back to DIVI theme. I have 756 products but it shows this. My shop is empty too. Was wondering why I'm getting this negative number for my products. Also is there a different theme that my help the sync process and speed because is super slow.
<img width="1440" alt="screen shot 2018-03-16 at 8 22 15 am" src="https://user-images.githubusercontent.com/37356040/37495991-6058357a-28f3-11e8-99d9-12d3104bfe75.png">

**Comment by hakeem1223** [comment]
Or is there anything I can modify or edit for the woocommerce for facebook plugin code in my cpanel?

**Comment by dmitridr** [comment]
Can you try re-setting entirely via `Reconfigure > Advanced Options > Delete`, then setting up again? This should be a more reliable and faster way of re-syncing rather than the Force Resync button, if that is what you are using here.

The negative items remaining is likely a bug, but resetting could fix it.

**Comment by hakeem1223** [comment]
Ok I reset everything. Will see what progress I make. Thanks.

**Comment by hakeem1223** [comment]
Now I got this:

Facebook extension error: Facebook for WooCommerce
Unsupported post request. Object with ID '162260627820896' does not exist, cannot be loaded due to missing permissions, or does not support this operation. Please read the Graph API documentation at https://developers.facebook.com/docs/graph-api

**Comment by dmitridr** [comment]
Most likely because you reset during an ongoing force-resync some item could have been deleted while in memory, it should be okay to ignore, provided that the reset actually worked and the products are uploaded this time.

**Comment by dmitridr** [comment]
I was looking into our logs, I saw a catalog created and items were uploading into it, but now I see it is deleted again :/ I think you may need to give the plugin some time after the reset (maybe a few minutes?) then you should be able to see the products in the shop. There should be no need to click `Force Resync`

**Comment by dmitridr** [comment]
The pics not showing which you mentioned earlier is actually normal for the period during which the shop and the products are under review (we can't show pics that don't pass review), this is normally up to 48 hours.

**Comment by dmitridr** [comment]
Some updates here : 

We changed the way `Force Resync` works in v1.8.2, if you upgrade and reset the plugin, `Force Resync` will always use a feed upload to resync, which is much faster and there should be no more strange negative numbers.

**Comment by vijaykumarim** [comment]
Same problem for me
